bot/core/events/handlers.go

```diff
@@ -57,12 +58,17 @@ func HandleGuildCreate(event *EventData) (err error) {
 		return
 	}
 
-	_, err = common.Session.ChannelMessageSendEmbed(config.GuildsChannel.GetInt64(), &discordgo.MessageEmbed{
+	embed := &discordgo.MessageEmbed{
 		Title:       "Guild Join",
 		Description: fmt.Sprintf("%s (%d)", guild.Name, guild.ID),
 		Color:       config.EmbedSuccessColor,
 		Timestamp:   time.Now().Format(time.RFC3339),
-	})
+	}
+	embed.Footer = &discordgo.MessageEmbedFooter{
+		Text: fmt.Sprintf("%s", strconv.Itoa(len(common.State.Guilds))),
```

Comment:
why are you formatting a string into itself?

Reply:
oh oops 🤦